### ISSUE TITLE
Some one know flask-bcrypt vs werkzeug.security

### ISSUE THREAD

**Issue by tim-hub** [maxcountryman/flask-bcrypt#42]
http://werkzeug.pocoo.org/docs/0.11/utils/#module-werkzeug.security

https://flask-bcrypt.readthedocs.io/en/latest/

which is safer?

**Comment by silpol** [comment]
former uses PBKDF while latter uses bcrypt

now "Why bcrypt is somewhat better than PBKDF2, Why bcrypt is not optimally secure, What NIST recommends" from [Sep'2010](http://security.stackexchange.com/a/6415/4936)

plus "SHA-256, in particular, benefits a lot from being implemented on a GPU. Thus, if you use SHA-256-crypt, attackers will be more at an advantage than if you use bcrypt, which is hard to implement efficiently in a GPU. ... Though SHA-256-crypt is not PBKDF2, it is similar enough in its performance behaviour on GPU, so the same conclusions apply."  from [Aug'2016](http://security.stackexchange.com/a/133251/4936) 

in other words, unless you serve US govt servants and hence must use FIPS / NIST rules, stick to bcrypt for time being.

**Comment by sheabae86** [comment]
Duplicate of #``
